apps/system/js/nfc_handover_manager.js

```diff
@@ -631,18 +609,26 @@
 
     /**
      * Check if a device is already paired and connected.
+     *
+     * BTv2 deprecate the device.connected property.
+     * connect stat should be retrieved by adapter.getConnectedDevices API
+     *
      * @param {Object} btssp BT SSP record
      * @memberof NfcHandoverManager.prototype
      */
     _checkConnected: function _checkConnected(btssp) {
-      if (!this.bluetooth.enabled) {
+      if (!Service.query('Bluetooth.isEnabled')) {
         this._onRequestConnect(btssp);
         return;
       }
       this._findPairedDevice(btssp.mac, (device) => {
-        if (!device.connected) {
-          this._onRequestConnect(btssp);
+        if (this._adapter === null) {
+          this._logVisibly('No Bluetooth Adapter');
+          return;
         }
+        this._adapter.getConnectedDevices(device).onerror = () => {
```

Comment:
I don't know why you have to do `getConnectedDevices()` here. I don't see the process in the old code base.

And the usage is in-corrected. The parameter is `serviceUuid`(profile ID).
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothAdapter/getConnectedDevices